lua/harpoon/buffer.lua

```diff
@@ -73,6 +73,20 @@ function M.setup_autocmds_and_keymaps(bufnr)
         "<Cmd>lua require('harpoon.buffer').run_select_command()<CR>",
         {}
     )
+    vim.api.nvim_buf_set_keymap(
+        bufnr,
+        "n",
+        "<C-x>",
```

Comment:
Any reason why this is `<C-x>` and not `<C-s>`? I feel like it should be consistent with default split binds as long as it's hardcoded.

Reply:
> why this is `<C-x>`

the only reason is https://github.com/ThePrimeagen/harpoon/blob/867e212ac153e793f95b316d1731f3ca1894625e/lua/harpoon/init.lua#L41

:D

Reply:
Ah I see! Nevermind then haha